### PR TITLE
comtrya 0.9.0 (new formula)

### PR DESCRIPTION
**Pull request by martintc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by daeho-ro** [comment]
Thank you for submitting a PR, @martintc .

You can check the style of the formula by 
```
brew style --fix comtrya
```
and also we need to add a test stanza for real use case of this library.

**Comment by martintc** [comment]
> Thank you for submitting a PR, @martintc .
> 
> You can check the style of the formula by
> 
> ```
> brew style --fix comtrya
> ```
> 
> and also we need to add a test stanza for real use case of this library.

Thank you for the feedback and I will get on top of this!

File: Formula/c/comtrya.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://github.com/comtrya/comtrya/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "a5401004a92621057dab164db06ddf3ddb6a65f6cb2c7c4208a689decc404ad4"
+  license "MIT"
+  head "https://github.com/comtrya/comtrya.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./app")
+  end
```
Comment (chenrui333) [inline Formula/c/comtrya.rb:13]: you also need test block for the formula

Comment (martintc) [reply Formula/c/comtrya.rb:13]: Added a test block, let me know if that is sufficient.

File: Formula/c/comtrya.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Comtrya < Formula
+  desc "Configuration and dotfile management tool"
+  homepage "https://comtrya/dev"
```
Comment (chenrui333) [inline Formula/c/comtrya.rb:3]:
```suggestion
  homepage "https://comtrya.dev"
```

**Review by chenrui333 — APPROVED** [review]
looks good after squashing the commits.

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**